src/openrct2/management/news_item.h

```diff
@@ -39,27 +39,22 @@ enum {
 
 extern const uint8 news_type_properties[10];
 
-#pragma pack(push, 1)
 /**
- * News item structure.
- * size: 0x10C
+ * A single news item / message.
  */
-typedef struct rct_news_item {
-	uint8 type;					// 0x00
-	uint8 flags;				// 0x01
-	uint32 assoc;				// 0x02
-	uint16 ticks;				// 0x06
-	uint16 month_year;			// 0x08
-	uint8 day;					// 0x0A
-	uint8 pad_0B;				// 0x0B
-	utf8 text[256];				// 0x0C
-} rct_news_item;
-assert_struct_size(rct_news_item, 12 + 256);
-#pragma pack(pop)
+typedef struct NewsItem {
+	uint8   Type;
+	uint8   Flags;
```

Comment:
We use flags everywhere else? What's the obsession with using a `boolean` now? Is it just so the expressions are simpler? In C++ we can always add a helper method to return a boolean representing the flag.